OpenJPEG 2.4 does not have the opj_decoder_set_strict_mode() function.

--- a/third_party/pdfium/core/fxcodec/jpx/cjpx_decoder.cpp
+++ b/third_party/pdfium/core/fxcodec/jpx/cjpx_decoder.cpp
@@ -492,11 +492,6 @@ bool CJPX_Decoder::Init(pdfium::span<con
     return false;
   }
 
-  // For https://crbug.com/42270564
-  if (!strict_mode) {
-    CHECK(opj_decoder_set_strict_mode(codec_.get(), false));
-  }
-
   opj_image_t* pTempImage = nullptr;
   if (!opj_read_header(stream_.get(), codec_.get(), &pTempImage)) {
     return false;
